index.html

```diff
@@ -44,14 +44,16 @@
           <h3 class="card-heading">Upcoming Events</h3>
           {% assign any_events = false %}
           {% for event in site.events %}
-          {% if event.date > site.time %}
+          {% comment %}{# Changing this condition? also change it in the below above #}{% endcomment %}
```

Comment:
Yes, I realise this is a bit odd to have two styles of comment. What's going on here is that `liquid` has comments as `{% comment %} ... {% comment %}` ([docs](https://shopify.github.io/liquid/tags/comment/)), but most editors won't do anything special with that.
However, many editors will highlight jinja2 style `{# ... #}` comments, so by putting that inside we get both comment behaviour and visual attention.

Reply:
`in the below above` seems pretty cryptic. Should it just be `in the below` or does the `above` refer to something like position in the rendered page? Either way think this comment needs to be a bit clearer

Reply:
8bf09ad46cd854cef02ee61331b300b658b219ce